templates/dashboard.html.twig

```diff
@@ -70,6 +70,12 @@
                                             target="_blank"
                                             v-if="recipe.repo.slug !== 'private'"
                                     ><v-icon>code</v-icon>&nbsp;&nbsp;Package</v-btn>
+                                    <v-btn
```

Comment:
I'd love to keep only one button for this purpose instead of creating two. How about outsourcing the `:href` to a method `getRepoPackagistUrl()` and the `v-if` to a method `showRepoPackagistButton()` and keep one single button for both official and private repos?

Reply:
Ok, i didnt know the code structure used in twig and i needed to try it quick. I'll prepare suggested changes and update PR later :)

Reply:
That's actually a vue component. Are you familiar with vue.js? I could clean that up if you were not. :-)

Reply:
Good to know :) I will try to learn something new :)